gxf_extensions/deltacast_videomaster/FindVideoMaster.cmake

```diff
@@ -14,9 +14,9 @@
 # limitations under the License.
 
 # Find the main root of the SDK
-find_path(VideoMaster_SDK_DIR NAMES Include/VideoMasterHD_Core.h PATHS /usr/local/deltacast REQUIRED)
+find_path(VideoMaster_SDK_DIR NAMES VideoMasterHD_Core.h PATHS /usr/include/videomaster REQUIRED)
 
-find_path(VideoMaster_INCLUDE_DIR NAMES VideoMasterHD_Core.h PATHS ${VideoMaster_SDK_DIR}/Include REQUIRED)
+find_path(VideoMaster_INCLUDE_DIR NAMES VideoMasterHD_Core.h PATHS ${VideoMaster_SDK_DIR} REQUIRED)
```

Comment:
Are these changes in reflection of a pathing update between library versions? Please note the required library version in app README instructions and `metadata.json` dependencies

Reply:
I have update the metadata dependencies to reflect that!